can/tests/test_can_parser.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+import glob
+import os
+import time
+import unittest
+
+from opendbc import DBC_PATH
+from opendbc.can.packer import CANPacker
+from opendbc.can.parser import CANParser
+from opendbc.can.tests.test_packer_parser import can_list_to_can_capnp
+from opendbc.generator.generator import generated_suffix
+
+
+class TestCANParser(unittest.TestCase):
+  @classmethod
+  def setUpClass(cls):
+    cls.dbcs = []
+    for dbc in glob.glob(f"{DBC_PATH}/*{generated_suffix}"):
+      cls.dbcs.append(os.path.basename(dbc).split('.')[0])
+
+  def test_detecting_dbcs(self):
+    self.assertGreaterEqual(len(self.dbcs), 20)
```

Comment:
this isn't a great test. either check that all the real DBCs are detected or create a tmp directory with a random number of fake DBCs